tests/integration/test_script_runner.py

```diff
@@ -26,14 +26,14 @@ class BackendDependencies(enum.Enum):
     {
         "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/database/bigquery_yaml_example.py",
         "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
-        "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples",
+        "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples/first_3_files",
```

Comment:
I've added this parameter to speed up the docs integration tests, which begin by copying over all the files from the directory specified in `data_dir`.  Currently the tests will only 1 file in the folder (this PR adds a tests that reads in 3 CSV files as a single Spark dataframe) but  will copy over all 36 files in the `taxi_yellow_trip_data_samples` directory.